internal/config/init.go

```diff
@@ -53,13 +55,30 @@ var InitCommand = &command.Command{
 		args []string,
 		globalFlags command.GlobalFlags,
 		services *services.Services,
+		proj *project.Project,
 	) (command.Result, error) {
-		proj, err := services.Project.Init(
+
+		sigAlgo := crypto.StringToSignatureAlgorithm(initFlag.ServiceKeySigAlgo)
```

Comment:
this code is also here https://github.com/onflow/flow-cli/pull/286/files#diff-e012f7fa2218d10a3c80f965cd049a7cfbfb2e7c89a3c5fdf4c8b2fbdc24b0ecR73. Could be a function...

Reply:
Not sure what do you mean?

Reply:
I thought that this part could be replaced with a library function, since its repeated a few times. But now that I look at it again, it wouldn't really make the method much shorter.

Reply:
Yeah, I had it extracted in lib function but then I realised I'm just putting another layer without really shortening the code